backend/spec/system/backoffice/investors_spec.rb

```diff
@@ -97,6 +97,18 @@
           end
         end
       end
+
+      context "when validation errors" do
+        before { approved_investor.account.update_attribute(:picture, nil) }
+
+        it "doesn't change the status" do
+          within_row(unapproved_investor.name) do
+            expect(page).to have_text(ReviewStatus.find("unapproved").name)
+            click_on t("backoffice.common.reject")
+            expect(page).not_to have_text(ReviewStatus.find("approved").name)
```

Comment:
maybe it would be also worthy to check that error message is displayed to user in this case